doc/users/next_whats_new/imshow_m_n_1.rst

```diff
@@ -0,0 +1,6 @@
+Imshow now coerces 3D arrays with depth 1 to 2D
+------------------------------------------------
+The number of ticks on colorbars was appropriate for a large colorbar, but
```

Comment:
The number of ticks on colorbars seems unrelated.

Reply:
sorry! I hadn't seen that from the copy paste